docs/aws/backend-deployment.md

```diff
@@ -0,0 +1,73 @@
+# AWS Backend Deployment
+
+
+## AWS Management Console
+ - Log in to the [AWS Management Console](https://us-east-2.console.aws.amazon.com/console/home?nc2=h_ct&region=us-east-2&src=header-signin#)
+   - If deploying to C4C infrastructure, select `IAM User` and enter account alias `c4cneu` and your *personal* username / password
+ - Select region `US East (Ohio)` (also called `us-east-2`)
+
+## RDS Setup
+ - Navigate to `RDS` from the `Services` dropdown and click `DB Instances`
+ - If creating a new RDS instance:
+   - Click `Create database`, then select `Standard create`, and `PostgreSQL`
+   - For `Templates`, select `Free tier`
+   - Name the RDS instance under `DB instance identifier`
+   - Set the `Master username` to `postgres` and enter a unique password
+   - Adjust `Maximum storage threshold` to a reasonable value (e.g. `100` GiB)
+   - Under `Connectivity`, set `Virtual private cloud (VPC)` to `VPC-Public`
+   - For `Subnet group`, select `db-sbg-public`
+   - For `Public access`, select `Yes`
+   - For `Existing VPC security groups` select `SG-RDS`
+   - Under `Database authentication options`, select `Password authentication`
+   - Adjust any other default settings on this page, as necessary
+   - Click `Create database`
+ - Connect to the RDS instance, and create a new database for the project
+
+## Elastic Beanstalk Setup
+ - Navigate to `Elastic Beanstalk` from the `Services` dropdown
+ - Click `Applications` from the left sidebar
```

Comment:
Do we not want a new application for each?

Reply:
I don't see any reason to have a new app for each